test/suites/projects.sh

```diff
@@ -561,6 +561,43 @@ test_projects_limits() {
 
   deps/import-busybox --project p1 --alias testimage
 
+  # Test per-pool limits.
```

Comment:
Please can we add tests for the hidden pools feature too.

Reply:
This is checked on lines 572-573